Code/GraphMol/Depictor/RDDepictor.h

```diff
@@ -77,7 +77,8 @@ RDKIT_DEPICTOR_EXPORT unsigned int compute2DCoords(
     RDKit::ROMol &mol, const RDGeom::INT_POINT2D_MAP *coordMap = nullptr,
     bool canonOrient = false, bool clearConfs = true,
     unsigned int nFlipsPerSample = 0, unsigned int nSamples = 0,
-    int sampleSeed = 0, bool permuteDeg4Nodes = false, bool forceRDKit = false);
+    int sampleSeed = 0, bool permuteDeg4Nodes = false, bool forceRDKit = false,
+    bool useRingTemplates = false);
```

Comment:
We really need a version of this function which takes a params object so that it's easier to change some of these arguments.

Reply:
Ok, I added a params object and named it `Compute2DCoordParameters` -- let me know if you have a better idea. I tried to make it distinguishable from `CoordGenParams`.